lib/rspecq/reporter.rb

```diff
@@ -83,6 +83,14 @@ def summary(failures, errors, flaky_jobs, duration)
       end
 
       summary = ""
+      if @queue.suite_failed?
+        summary << "\n\n"
+        summary << "**Fail fast was enabled for this run**(threshold=#{@queue.fail_fast_threshold})"
+        summary << "\n"
+        summary << "The test run has quited after #{@queue.fail_fast_threshold} failed tests."
+        summary << "\n"
+      end
```

Comment:
Could you please add a sample output in the PR description, so that we can understand how this looks to the user?

Reply:
done